.github/workflows/simple-pytest-cli.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
There's already a CLI tests workflow: `pytest-cli.yml`.
Not sure if we've either been unclear about that there are tests run already in the CLI, or if this is for the gui somehow?

Reply:
Decided in meeting: this can be removed and the `pytest-cli.yml` will be changed to run on push for all branches.